openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -496,8 +587,18 @@ def response_to_actions(response: ModelResponse) -> list[Action]:
                 ) from e
             if tool_call.function.name == 'execute_bash':
                 action = CmdRunAction(**arguments)
-            elif tool_call.function.name == 'execute_replay':
-                action = ReplayCmdRunAction(**arguments)
+            elif tool_call.function.name == 'inspect-data':
+                action = ReplayToolCmdRunAction(
+                    command_name='inspect-data',
+                    command_args=arguments | {'recordingId': state.replay_recording_id},
```

Comment:
NOTE: We provide the `recording_id` that we gathered from our `initial-analysis`. The agent does not need to know about it.